### PR TITLE
Feature/pelagos 3676 remove assetic

### PR DESCRIPTION
**Pull request by mickel1138**
Sorry, so MANY files...

### CODE REVIEW

**Comment by mickel1138** [comment]
> fine uploader breaks on dataset-review application

Yes, I see now, I forgot fineuploader.js

File: src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/projects.html.twig [hunk 1]
```diff
@@ -5,67 +5,61 @@
...
+                    <td>
+                        <div class="dotchart_legend" style="position: relative;">
+                            <span id="a"><strong>Dataset with status:</strong>
+                                <img src="{{ asset('build/images/green-circle.png') }}"> Available:
+                            </span>
+                            <span id="r">
+                                <img src="{{ asset('build/images/yellow-circle.png') }}"> Submitted:
+                            </span>
+                            <span id="i">
+                            {% image '@PelagosAppBundle/Resources/public/images/empty-circle.png' %}
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/projects.html.twig]: You missed a spot :D

File: src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSubmission/qtip.html.twig [hunk 2]
```diff
@@ -1,63 +1,49 @@
 
 <div id="regid_tip" style="display:none;">
-    {% image '@PelagosAppBundle/Resources/public/images/info.png' %}
-    <img class="info" src="{{ asset_url }}"  style="float:right;">
-    {% endimage %}
+    <img class="info" src="{{ asset('build/images/info32.png') }}">
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSubmission/qtip.html.twig]: do you want to add the style=float right back?

File: src/Pelagos/Bundle/AppBundle/Resources/views/List/FundingOrganizations.html.twig [hunk 3]
```diff
@@ -1,38 +1,23 @@
...
 
-    {{
-        add_css (
-            [
-                '//cdn.datatables.net/1.10.7/css/jquery.dataTables.min.css',
-                '//cdn.datatables.net/select/1.0.1/css/select.dataTables.min.css'
-            ]
-        )
-    }}
+{%block stylesheets %}
```
Comment (praneethpr) [inline src/Pelagos/Bundle/AppBundle/Resources/views/List/FundingOrganizations.html.twig]: I think there are few more places which needs the space

File: src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/projects.html.twig (hunk 1 above)
Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/projects.html.twig]: Good find!

File: src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/datasets.html.twig [hunk 4]
```diff
@@ -35,25 +35,11 @@
...
-
-        {% image '@PelagosAppBundle/Resources/public/images/x.png' %}
-        {% set red_icon = asset_url %}
-        {% endimage %}
-
-        {% image '@PelagosAppBundle/Resources/public/images/na.png' %}
-        {% set na_icon = asset_url %}
-        {% endimage %}
+        {% set green_icon = asset('build/images/check.png') %}
+        {% set green_remote_icon = asset('build/images/check-remote.png.png') %}
```
Comment (fightingtexasaggie) [inline src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/datasets.html.twig]: .png.png? ;-)

Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/DatasetMonitoring/datasets.html.twig]: Who's there!?

File: src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSubmission/qtip.html.twig (hunk 2 above)
Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSubmission/qtip.html.twig]: I almost want to make this qtip into it's own template. But I think most have the style added to the class.

Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSubmission/qtip.html.twig]: I added it, but doesn't seem to make a difference.

File: src/Pelagos/Bundle/AppBundle/Resources/views/template/forms/DataRepository.html.twig [hunk 5]
```diff
@@ -9,11 +9,6 @@ _action="{{path('pelagos_api_data_repositories_get_collection')}}">
     {{ form_widget(form) }}
 
     <div class="innerForm">
-        {% if is_granted('CAN_EDIT', DataRepository) %}
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/template/forms/DataRepository.html.twig]: These should not be removed (if block)

**Comment by mickel1138** [comment]
> Dataset Review Erddap URL validation seems to not be working now.

@fightingtexasaggie is this stil an issue?

**Pull request merged**